test/commands/commit/promote.test.ts

```diff
@@ -46,4 +76,26 @@ describe("commands/commit/promote", () => {
         );
       });
   });
+
+  describe("given an `only` commit ID flag", () => {
+    setupWithStubPromoteCommit()
+      .stdout()
+      .command(["commit promote", "--only", "example-id"])
+      .it("calls apiV1 promoteChange with expected props", () => {
+        sinon.assert.calledWith(
+          KnockApiV1.prototype.promoteChange as any,
+          sinon.match(({ flags }) =>
+            isEqual(flags, {
+              "service-token": "valid-token",
+              only: "example-id",
+            }),
+          ),
+          sinon.match((commit) =>
```

Comment:
I don't know why this test fails, I checked and the `commit` is returned when calling `promoteChange` 🧐